examples/extract_data/src/extract_data_login.py

```diff
@@ -70,67 +70,69 @@ async def run():
             print("No profileId provided")
             profile_id = None
 
-        create_session_response = await client.sessions.create({
-            "configuration": {
-                "timeoutMinutes": 10,
-                "persistProfile": not profile_id,
-                "baseProfileId": profile_id,
-            }
-        })
+        if not session.data.cdp_ws_url:
```

Comment:
session is not defined at this point. This code should go after you create the session.

This throws an error:
![image](https://github.com/user-attachments/assets/75b96b9c-45c1-4c3a-a45c-cde3329a1bdd)

Reply:
Replaced this code in the right place, it's working now